src/Symfony/Component/HttpClient/Tests/TraceableHttpClientTest.php

```diff
@@ -80,4 +82,21 @@ public function testItResetsTraces()
         $sut->reset();
         $this->assertCount(0, $sut->getTracedRequests());
     }
+
+    /**
+     * @requires extension curl
+     */
+    public function testStream()
+    {
+        TestHttpServer::start();
+
+        $sut = new TraceableHttpClient(new CurlHttpClient());
```

Comment:
and now we can use `NativeHttpClient` :)

Reply:
Arf sorry ! Fixed